sympy/assumptions/facts.py

```diff
@@ -34,71 +36,138 @@ def get_composite_predicates():
 
 @cacheit
 def get_known_facts():
+    """
+    Facts between unary predicates.
+
+    Returns
+    =======
+
+    x : Placeholder symbol for unary facts.
+
+    fact : Known facts in conjugated normal form.
+
+    """
     # We build the facts starting with primitive predicates.
     # DO NOT include the predicates in get_composite_predicates()'s keys here!
-    return And(
-
+    x = Symbol('x')
+    fact = And(
         # primitive predicates exclude each other
-        Implies(Q.negative_infinite, ~Q.positive_infinite),
-        Implies(Q.negative, ~Q.zero & ~Q.positive),
-        Implies(Q.positive, ~Q.zero),
+        Implies(Q.negative_infinite(x), ~Q.positive_infinite(x)),
+        Implies(Q.negative(x), ~Q.zero(x) & ~Q.positive(x)),
+        Implies(Q.positive(x), ~Q.zero(x)),
 
         # build real line and complex plane
-        Implies(Q.negative | Q.zero | Q.positive, ~Q.imaginary),
-        Implies(Q.negative | Q.zero | Q.positive | Q.imaginary, Q.algebraic | Q.transcendental),
+        Implies(Q.negative(x) | Q.zero(x) | Q.positive(x), ~Q.imaginary(x)),
+        Implies(Q.negative(x) | Q.zero(x) | Q.positive(x) | Q.imaginary(x),
+                Q.algebraic(x) | Q.transcendental(x)),
 
         # other subsets of complex
-        Implies(Q.transcendental, ~Q.algebraic),
-        Implies(Q.irrational, ~Q.rational),
-        Equivalent(Q.rational | Q.irrational, Q.negative | Q.zero | Q.positive),
-        Implies(Q.rational, Q.algebraic),
+        Implies(Q.transcendental(x), ~Q.algebraic(x)),
```

Comment:
This implication is in both directions so it should be
```python
Not(Q.transcendental(x) & Q.algebraic(x))
```

Reply:
`Implies(a, ~b)` and `Not(a & b)` are logically equivalent. I guess the latter might be easier to reason about. I'm +0 to changing these facts like this. to_cnf should create the same thing at the end of the day for both. 

Note that this isn't quite the same as an implication in both directions. `Implies(a, ~b)` and `Implies(a, ~b)` are equivalent by contrapositive, but `a iff ~b` (`Equivalent(a, ~b)`) is a stronger statement. It doesn't hold here (and in similar cases) because x might be neither algebraic nor transcendental if it isn't a complex number.

Reply:
Also given that the actual facts are compiled with ask_update, it doesn't really matter what form of an expression we use, or if we are redundant with facts. Redundancy might be potentially confusing, but it wouldn't slow things down because of this.

Reply:
Yes, I suppose they are equivalent. It seems that a number of facts come into this category where what you want to say is sort of like
```
complex  ==  transcendental xor algebraic 
```
I'm wondering what is the best way to write that so that it's completely clear and clearly symmetric. Maybe:
```
Equivalent(complex, algebraic | transcendental)
Not(algebraic & transcendental)
```
It feels like there should be a simpler way to say this in one somehow.